lib/version_utils.pm

```diff
@@ -599,12 +603,13 @@ Default to I</etc/os-release>.
 Returns 1 (true) if the ID_LIKE variable contains C<distri_name>.
 
 =cut
-sub check_host_os {
-    my ($distri_name, $os_release_file) = @_;
-    die "$distri_name is not given" unless $distri_name;
+sub check_os_release {
+    my ($distri_name, $line, $os_release_file) = @_;
+    die '$distri_name is not given' unless $distri_name;
+    die '$line is not given' unless $line;
     $os_release_file //= '/etc/os-release';
-    my $os_like_name = script_output("grep -e \"^ID_LIKE\\b\" ${os_release_file} | cut -c4- | tr -d '\"'");
-    return ($os_like_name =~ /$distri_name/);
+    my $os_like_name = script_output("grep -e \"^$line\\b\" ${os_release_file} | cut -c4- | tr -d '\"'");
```

Comment:
you have one small problem here. `grep -e \"^PRETTY_NAME\b\" ${os_release_file} | cut -c4-  | tr -d '"'` will return `TTY_NAME=<pretty_name>`. you have to count the len of $line and pass it in place of 4

Reply:
I did this to solve it:
```perl
my $os_like_name = script_output("grep -e \"^$line\\b\" ${os_release_file} | cut -d'\"' -f2");
```